tests/core/propertyTypes.test.js

```diff
@@ -156,6 +156,26 @@ suite('propertyTypes', function () {
     });
   });
 
+  suite('matrix4', function () {
+    var parse = propertyTypes.matrix4.parse;
+    var stringify = propertyTypes.matrix4.stringify;
+
+    test('parses matrix4', function () {
```

Comment:
@ngokevin resolving a THREE.js object sounds handy, but is inconsistent with the vector types. Compatibility with THREE.Matrix4.prototype.toArray() and fromArray() seems critical.

(that said, it would be neat for all vectors and matrices to resolve as THREE types, but outside scope here)